CHANGELOG.md

```diff
@@ -60,6 +61,17 @@ Adding a new version? You'll need three changes:
  - [0.0.5](#005)
  - [0.0.4 and prior](#004-and-prior)
 
+## [2.8.0]
+
+> Release date: TBD
+
+### Fixed
+
+- When the KongState is built, if there are Kubernetes service that have
+  inconsistent annotations, the an error message is logged and the service
+  is skipped, instead of breaking the whole configuration.
+  [#2942](https://github.com/Kong/kubernetes-ingress-controller/pull/2988)
```

Comment:
```suggestion
- When the KongState is built, if there are Kubernetes services that have
  inconsistent annotations, an error message is logged and the service
  is skipped, instead of breaking the whole configuration.
  [#2942](https://github.com/Kong/kubernetes-ingress-controller/pull/2988)
```